multigeiger/version.h

```diff
@@ -1,11 +1,16 @@
 // Revision history:
 
-const char* revString = "V1.10_2019-12-13";
-#define SOFTWARE_VERSION 1A
+const char* revString = "V1.11_2019-12-16";
```

Comment:
the date is not really needed here, people can look it up in the changelog, if interested.

also, we could get rid of the "V" and just use "1.11". would also make tagging in git easier and checking out some specific tag.

Reply:
when changing like i suggested, we do not need revString at all. it could be just computed from SOFTWARE_VERSION.

Reply:
We want to show the date on the start screen, so we need it.
I will separate version and date and calculate the display string from that.

Reply:
I just add SOFTWARE_DATE to already existing SOFTWARE_VERSION and compute all from these.

Reply:
Or just add SOFTWARE_DATE and compute all from SOFTWARE_*.